bigtable/instanceadmin.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Import google bigtable client lib
+require "google/cloud/bigtable"
+
+def create_prod_instance project_id, instance_id, cluster_id
+  bigtable = Google::Cloud::Bigtable.new project_id: project_id
+  puts "Check Instance Exists"
+
+  # [START bigtable_check_instance_exists]
+  if bigtable.instance instance_id
```

Comment:
Add comments explaining what the method arguments are, similar to https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/44f143a7dc30b61a65fd5611e6fca6002cc8a67b/vision/detect_text.rb#L19 .